docker-compose.yml

```diff
@@ -141,6 +141,7 @@ services:
       SLACK_WEBHOOK_URL: ${SLACK_WEBHOOK_URL}
       ORDNANCE_SURVEY_API_KEY: ${ORDNANCE_SURVEY_API_KEY}
       MINIO_PORT: ${MINIO_PORT}
+      CORS_ALLOWLIST: ${EDITOR_URL_EXT}
```

Comment:
As this is just a "derived" env var, we actually don't need to add it to the `.env` files and we can just populate it here.

If this is unclear / unhelpful I can add the the `.env` and `.env.example` files.